test/module.test.js

```diff
@@ -1762,3 +1762,92 @@ describe('generate with prefix strategy', () => {
     expect(canonicalLink?.getAttribute('href')).toBe('nuxt-app.localhost/en')
   })
 })
+
+describe('Configuration with locale fallback decision map', () => {
+  /** @type {Nuxt} */
+  let nuxt
+
+  const locales = [
+    {
+      code: 'es',
+      iso: 'es',
+      name: 'Spanish',
+      file: 'es.js'
+    },
+    {
+      code: 'en-GB',
+      iso: 'en-GB',
+      name: 'Britain english',
+      file: 'en-GB.js'
+    },
+    {
+      code: 'fr-FR',
+      iso: 'fr-FR',
+      name: 'Français',
+      file: 'fr-FR.js'
+    },
+    {
+      code: 'de',
+      iso: 'de',
+      name: 'Deutsch',
+      file: 'de.js'
+    }
+  ]
+
+  afterEach(async () => {
+    await nuxt.close()
+  })
+
+  test('should be fallback translation by respecting the decision map', async () => {
+    const override = {
+      i18n: {
+        lazy: true,
+        langDir: 'lang/',
+        locales,
+        vueI18n: {
+          fallbackLocale: {
+            'fr-FR': ['en-GB', 'es'],
+            default: ['de']
+          }
+        }
+      }
+    }
+
+    const localConfig = loadConfig(__dirname, 'locale-decision-map-fallback', override, { merge: true })
+    nuxt = (await setup(localConfig)).nuxt
```

Comment:
I'd prefer to follow the existing convention and have fixture loading in `beforeAll` hooks. Then you just need to have separate `describe` blocks for your tests.

This is better because it allows adding more tests in those describe blocks without having to re-build the fixture for each test.

Reply:
You're right, I've fixed it.